### DISCUSSION TITLE
is there any way to pass 1 variable to getHeaderWidgets()

### DISCUSSION THREAD

**Discussion by phamduc1823** [filamentphp/filament#7998] [General]
hello everyone, is there a way to pass a variable to getHeaderWidgets() and retrieve it in the corresponding class?

**Comment by danharrin** [comment]
What are you trying to do.

**Reply by phamduc1823** [reply]
Thank you for your interest, I have solved it, I want to pass a value to it to retrieve the data in the class, my code:

    protected function getHeaderWidgets(): array
    {
        return [
            UserInformationWidget::make(
                [
                    'user' => $this->user,
                ]
            ),
        ];
    }
    sorry my english is not good maybe i misinterpreted my problem
